app/move/fields/create.js

```diff
@@ -344,17 +344,44 @@ module.exports = {
   },
   date_from: {
     ...dateField,
-    validate: [...dateField.validate, 'required', 'after'],
+    validate: [...dateField.validate, 'after'],
     id: 'date_from',
     name: 'date_from',
     label: {
       text: 'fields::date_from.label',
       classes: 'govuk-label--s',
     },
   },
+  has_date_to: {
+    validate: 'required',
+    component: 'govukRadios',
+    name: 'has_date_to',
+    fieldset: {
+      legend: {
+        text: 'fields::has_date_to.legend',
+        classes: 'govuk-label--m',
+      },
+    },
+    items: [
+      {
+        value: 'yes',
```

Comment:
Maybe `true` / `false`? I know we need to use strings for the field values to stay populated but feels nicer to use true/false rather than yes/no as true/false values.

Reply:
Personally I find `yes/no` to be more human readable ¯\_(ツ)_/¯

Reply:
I'll keep yes / no for now if that's ok. 
The reason being: we don't have uniformity at the moment. We use an ample choice of booleans, `'false'` as string, `'no'` etc. 
It would be a nice thing to make it uniform. Maybe on next refactoring Friday?